bindata/network/openshift-sdn/000-ns.yaml

```diff
@@ -9,3 +9,16 @@ metadata:
   annotations:
     openshift.io/node-selector: "" #override default node selector
     openshift.io/description: "OpenShift SDN components"
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  # NOTE: this namespace is the portal for host network traffic
+  name: openshift-host-network
+  labels:
+    openshift.io/run-level: "0"
+    openshift.io/cluster-monitoring: "true"
```

Comment:
I don't know what that label does but we probably don't need it for this

Reply:
We would need this label if there are ServiceMonitor objects created in this namespace, which there won't be.